src/components/Contacts/ContactFilters/ContactFilters.mock.tsx

```diff
@@ -0,0 +1,59 @@
+import { MockedResponse } from '@apollo/client/testing';
+import {
+  ContactFiltersDocument,
+  ContactFiltersQuery,
+  ContactFiltersQueryVariables,
+} from '../../../../pages/accountLists/[accountListId]/ContactFilters.generated';
+import { gqlMock } from '../../../../__tests__/util/graphqlMocking';
+
+export const ContactFiltersQueryDefaultMocks = (
+  accountListId: string,
+): MockedResponse[] => {
+  const contactFiltersQuery = gqlMock<
+    ContactFiltersQuery,
+    ContactFiltersQueryVariables
+  >(ContactFiltersDocument, { variables: { accountListId } });
+
+  return [
+    {
+      request: {
+        query: ContactFiltersDocument,
+        variables: {
+          accountListId,
+        },
+      },
+      result: {
+        data: contactFiltersQuery,
+      },
+    },
+  ];
+};
+
+export const ContactFiltersQueryEmptyMocks = (
+  accountListId: string,
+): MockedResponse[] => {
+  return [
+    {
+      request: {
+        query: ContactFiltersDocument,
+        variables: {
+          accountListId,
+        },
+      },
+      result: {
+        data: { contactFilters: [] },
+      },
+    },
+  ];
+};
+
+export const ContactFiltersQueryLoadingMocks = (
+  accountListId: string,
+): MockedResponse[] => {
+  return [
+    {
+      ...ContactFiltersQueryDefaultMocks(accountListId)[0],
+      delay: 100931731455,
```

Comment:
What is this? You've got several of them.

Reply:
I didn't know how to create a mock loading state, but I found this in one of the other files: https://github.com/CruGlobal/mpdx-react/blob/main/src/components/Dashboard/ThisWeek/WeeklyActivity/WeeklyActivity.mock.ts#L110.  From what I can tell, it delays the results from actually returning, so when you test with Jest or Storybook it will still render in a loading state.  There's probably a better way to do that though.

Reply:
Is this part of Apollo's MockedProvider API? I haven't played with it... Are you having trouble getting it out of the loading state or keeping it in the loading state?

Reply:
Yeah I'm having trouble keeping it in a loading state... I hope there's a better way than this